### PR TITLE
Add more details to mantrid stats

### PR DESCRIPTION
**Pull request by emate**
Append info about blacklisted backends to mantrid's stats page.

### CODE REVIEW

**Comment by knowak** [comment]
lgtm

File: mantrid/actions.py [hunk 1]
```diff
@@ -31,6 +31,9 @@ def __init__(self, balancer, host, matched_host):
     def handle(self, sock, read_data, path, headers):
         raise NotImplementedError("You must use an Action subclass")
 
+    def blacklisted_backends():
```
Comment (knowak) [inline mantrid/actions.py]: self?

Comment (emate) [inline mantrid/actions.py]: of course :cry:

**Comment by karol-nowak** [comment]
Hm, why was this closed? Interference with monitoring tools?

**Comment by emate** [comment]
Performance degradation. It should probably be done with generators, but it wasn't worth re-implementing and testing at the time.

**Pull request closed without merging**